Shared/Extensions/URLExtensions.swift

```diff
@@ -316,6 +316,11 @@ extension URL {
         }
         return urls[0] == urls[1]
     }
+    
+    public var isFxHomeUrl: Bool {
+        return absoluteString.hasPrefix("internal://")
+    }
```

Comment:
@nishant2718 do you want to check all local urls as this `internal://` prefix will give you that and not just home

If you really want to check home then you need to follow something like `URL(string: "internal://local/about/home")`

Reply:
@nbhasin2, then why are we doing on here like this: https://github.com/mozilla-mobile/firefox-ios/blob/204d1e5e08e3cfe71ff5583a0288123318c45ca5/Client/Frontend/Browser/Tab.swift#L195 ? 

Does that mean that check should be `URL(string: "internal://local/about/home")` too ? Seems there's no consistent way in place.

Reply:
Discussed with Nish! Now I understand the purpose of `internal://` and  `internal://local/about/home`. Although I still think in the future we should standardize how we do both!